mlir/test/Conversion/AIRToAIE/air_channel_to_objectfifo_L1toL2.mlir

```diff
@@ -0,0 +1,74 @@
+//===- air_channel_to_objectfifo_L1toL2.mlir --------------------*- MLIR -*-===//
+//
+// Copyright (C) 2022, Advanced Micro Devices, Inc.
+// SPDX-License-Identifier: MIT
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: air-opt %s -air-place-herds='num-rows=2 num-cols=2 row-anchor=3 col-anchor=5' --air-to-aie='emit-while-loop=false row-offset=3 col-offset=5 use-objectfifo=true device=xcve2802' --canonicalize | FileCheck %s
```

Comment:
Can this command line be simplified to only `air-to-aie`?  `emit-while-loop=false` is already the default

Reply:
Good point. I'll take a look and simplify.